cronback-proto/project_srv.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+import "common.proto";
+
+package project_srv_proto;
+
+service ProjectService {
+  rpc CreateProject (CreateProjectRequest) returns (CreateProjectResponse);
+  rpc GetProjectStatus(GetProjectStatusRequest) returns (GetProjectStatusResponse);
+  rpc SetProjectStatus(SetProjectStatusRequest) returns (SetProjectStatusResponse);
+  rpc IsProjectExists(IsProjectExistsRequest) returns (IsProjectExistsResponse);
+}
+
+message CreateProjectRequest {}
+
+message CreateProjectResponse {
+  common.ProjectId id = 1;
+}
+
+message GetProjectStatusRequest {
+  common.ProjectId id = 1;
+}
+
+message GetProjectStatusResponse {
+  ProjectStatus status = 1;
+}
+
+message SetProjectStatusRequest {
+  common.ProjectId id = 1;
+  ProjectStatus status = 2;
```

Comment:
I'm assuming this is the previous status, can you leave a comment about this, or rename to make it clearer—`old_status`?

Reply:
hmmm, what do you mean by `old_status`? This is a `set` request, so it's the requested new value.

Reply:
Sorry, I meant to put this comment on the `SetProjectStatusResponse`.